iati_dashboard/settings.py

```diff
@@ -31,9 +31,28 @@
 
 DEBUG = env("DEBUG")
 
+SENTRY_DSN = env("SENTRY_DSN")
```

Comment:
You also need to add an entry to the instantiation of environ.Env above. This will make sure we don't get an error when the environment variable isn't declared.

Reply:
@Bjwebb I have added 'None' as a default value. Thanks.